### DISCUSSION TITLE
Exclude searching inside attachments of a document Library

### DISCUSSION THREAD

**Discussion by mrmarshallmark** [microsoft-search/pnp-modern-search#3308] [Q&A] [answered]
Can I configure PHP Search box to return results that do not search inside the attachments inside a document? Example: only search the TITLE of a SharePoint List.  Thank you. Mark

**Comment by wobba** [comment, marked as answer]
Use something like `Title:({searchTerms})`

**Reply by kasperbolarsen** [reply]
And perhaps add isDocument:1

**Reply by wobba** [reply]
Only for (most) Office formats and PDF's - so use with caution depending on your content.

**Comment by mrmarshallmark** [comment]
That works. Thanks.
